draft-ietf-quic-transport.md

```diff
@@ -3981,6 +3979,12 @@ A client MUST NOT include an original connection ID, a stateless reset token, or
 a preferred address.  A server MUST treat receipt of any of these transport
 parameters as a connection error of type TRANSPORT_PARAMETER_ERROR.
 
+max_connection_ids (0x000e):
+
+: The maximum number of connection IDs that the peer is willing to store.
+  This value includes only connection IDs sent in NEW_CONNECTION_ID frames.
```

Comment:
The maximum number of connection IDs that this endpoint can store for future use. These connection IDs are issued by the peer and sent in NEW_CONNECTION_ID frames.

Reply:
I *can* store a lot. But I might not be willing to.

Reply:
You have the two sides mixed up. This param should convey the max that _this_ endpoint is willing to store. For clarity, I also suggest that you explicitly call out who is sending the new connection IDs: "... includes only connections IDs sent by the peer..."

Reply:
I think what @janaiyengar is pointing out is that "the _peer_ is willing to store" needs to be changed to "the _endpoint_ is willing to store".

Reply:
Isn't the number of stored connection IDs symmetric? In practice, each address change requires both sides to change their CIDs (except for zero-length cases).

I'm not sure I see any point in ever storing more than min(my_max_connection_ids, your_max_connection_ids)

Reply:
The need for reciprocity in most cases that consume a connection ID does tend to lead to the numbers being approximately the same.  But it's not symmetric in the sense that they need to be equal.  The design is therefore asymmetric.  

If you consider a client that is moving between networks of questionable quality, they can burn connection IDs on futile probes, leading to greater consumption of connection IDs.  That suggests that a client might store more spares than a server.